containers/map-api/package.json

```diff
@@ -15,12 +15,13 @@
     "url": "https://github.com/IBM/secret-map-dashboard"
   },
   "dependencies": {
+    "assert": "^1.4.x",
     "body-parser": "^1.17.x",
-    "mongoose": "^4.13.x",
+    "cors": "^2.8.x",
     "express": "^4.15.x",
     "fs": "~0.0.1",
-    "assert": "^1.4.x",
-    "cors": "^2.8.x"
+    "mongoose": "^4.13.x",
+    "router": "^1.3.2"
   },
```

Comment:
we might need to incorporate some unit testing such as mocha. Some simple test cases could be like we test whether we receive a 200 or 201 status code 🥇 . something simple like that.

Reply:
Will look into mocha. Thanks!